### PR TITLE
Added Differencify

### PR DESCRIPTION
**Pull request by NimaSoroush**
Added Differencify to Tools and frameworks section and a blog post for how to use the tool

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,7 +95,8 @@ Finally, I'm sure everyone who reads this list has one thing they want to add. P
 - [Phillip Gourley: Making visual regression useful](https://medium.com/@philgourley/making-visual-regression-useful-acfae27e5031) - Why you should use BackstopJS.
 - [Pavels Jelisejevs: Visual Regression Testing with PhantomCSS](https://www.sitepoint.com/visual-regression-testing-with-phantomcss) - Introduction to PhantomCSS.
 - [Chromeless, Chrominator, Chromy, Navalia, Lambdium, GhostJS, AutoGCD](https://medium.com/@kensoh/chromeless-chrominator-chromy-navalia-lambdium-ghostjs-autogcd-ef34bcd26907) - Headless Chrome is shaking up traditional approaches to test automation.
-- [Visual regression testing using Jest, Chromeless and AWS Lambda](https://novemberfive.co/blog/visual-regression-testing-jest-chromeless-lambda) - Tutorial using Chromeless and jest-image-snapshot.
```
Comment (mojoaxel) [inline README.md]: Why did you remove this entry?
Can you please re-add it please.

Comment (NimaSoroush) [reply README.md]: @mojoaxel : That was a mistake. Updated

**Comment by NimaSoroush** [comment]
@mojoaxel : This should be good to go

**Comment by NimaSoroush** [comment]
Thanks @mojoaxel
